pages/common/texdoc.md

```diff
@@ -0,0 +1,16 @@
+# texdoc
+
+> Search for appropriate documentation for (La)TeX commands or packages.
+> More information: <https://texdoc.org/index.html>.
+
+- Open first search result in the default PDF viewer:
+
+`texdoc {{search}}`
+
+- List best search result:
```

Comment:
Perhaps this will read better?
```suggestion
- List the best search results:
```

`List` implies multiple items here. If it doesn't display multiple items, then maybe `Show the best search result` would be better?

Reply:
It should be `results`, I made a typo